index.tsx

```diff
@@ -975,6 +974,7 @@ export function compiler(
 
   const createElementFn = options.createElement || React.createElement
 
+  // JSX custom pragma
```

Comment:
Not adding `// noinspection JSUnusedLocalSymbols` since it'd probably break the eslint override, but at least explaining what's going on